syntax = "proto3";
option cc_generic_services = true;

import "http_rule.proto";

message AddQuery {
    int32 a = 1;
    int32 b = 2;
}

message AddResponse {
    int32 result= 1;
}

//service CalculateService {
//    rpc Add(AddQuery) returns (AddResponse);
//}

service CalculateService {
    rpc Add(AddQuery) returns (AddResponse) {
        option (tinyRPC.api.http) = {
            method: "post"
            url: "/v1/im/register_device"
        };
    }
}